supabase/migrations/20260816202033_restrict_app_versions_deleted_at.sql

```
@@ -0,0 +1,60 @@
+    NEW.app_id,
+    NULL::bigint
+  ) THEN
+    RAISE EXCEPTION 'PERMISSION_DENIED_BUNDLE_DELETE'
+      USING ERRCODE = '42501';
+  END IF;
+
+  RETURN NEW;
+END;
+$$;
+
+ALTER FUNCTION public.enforce_app_versions_delete_permission() OWNER TO postgres;
```

SQLFluff (4.2.2)

[error] Line is too long (81 > 80).

(LT05)